src/components/Header.js

```diff
@@ -0,0 +1,23 @@
+import React from 'react';
+import { Link } from 'react-router-dom';
+
+const Header = () => (
+  <header>
+    <nav>
+      <h1>Bookstore CMS</h1>
+      <ul>
+        <li>
+          <Link to="/">Books</Link>
+        </li>
+        <li>
+          <Link to="/categories">Categories</Link>
+        </li>
+      </ul>
+    </nav>
+    <div id="userAccount">
+      <i className="fa-solid fa-user" />
+    </div>
+  </header>
```

Comment:
- [ ] Kindly make sure your header sticks as much as possible to the template design. Right now your design misses the user icon. For that, I suggest you use `react-icons` as the one you used doesn't work. **[Here](http://react-icons.github.io/react-icons/search?q=fauser)** is how you can find the documentation on how to install and even use them.
### Template design
![Screenshot from 2022-11-17 21-25-36](https://user-images.githubusercontent.com/83042742/202543542-74b81ede-25bb-40cf-9bc4-415919075380.png)

### Your Design
![Screenshot from 2022-11-17 21-26-52](https://user-images.githubusercontent.com/83042742/202543602-7ed5f6f4-cdfb-419d-b5c9-b8ecb8813a5d.png)